src/Order.cs

```diff
@@ -70,5 +70,34 @@ public Order Edit(Dictionary<string, object> data)
                 return null;
             }
         }
+
+        public Order ViewRtoReview(Dictionary<string, object> data=null)
+        {
+            string relativeUrl = string.Format("{0}/{1}/{2}/rto_review", GetUrlVersion(), GetEntityUrl(), this["id"]);
+            List<Entity> entities = Request(relativeUrl, HttpMethod.POST, data);

```

Comment:
Is this a POST request?

Reply:
Yes it is a post request

<img width="729" alt="Screenshot 2024-05-06 at 1 26 12 PM" src="https://github.com/razorpay/razorpay-dot-net/assets/89454448/468129c4-8901-4569-89ff-eca7cf8a208e">